README.md

```diff
@@ -96,6 +102,20 @@ if (closed_issues.length) {
 }
 ```
 
+### Direct usage (not as a plugin)
```

Comment:
i based this addition on https://github.com/octokit/plugin-create-or-update-text-file.js/tree/main#direct-usage-not-as-plugin, but open to alternative suggestions if you think something else would fit this readme better

Reply:
That's good 👍🏼